<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoApplicationClass</class>
 <widget class="QMainWindow" name="DemoApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日常应用</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>251</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日常应用</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="dealImage">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>图像处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dealAudio">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>音频处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dealVideo">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>170</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>视频处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dealGeo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>几何处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dealOther">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>便捷工具</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dealConnect">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>430</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>远程连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DemoApplication.qrc"/>
 </resources>
 <connections/>
</ui>
